tests/README.md

```diff
@@ -179,6 +179,32 @@ typewriter = ["Test Font", "Test Font Fallback"]
 japanese_gothic = ["Test Font", "Test Font Fallback"]
 japanese_gothic_mono = ["Test Font", "Test Font Fallback"]
 japanese_mincho = ["Test Font", "Test Font Fallback"]
+
+# Support for multiple configurations (see the 'Multiple tests' section for more details)
+# Fields specified here will override the 'default' values provided in the rest of the document.
+[configs.CONFIG_NAME]
```

Comment:
Nit: what do you think about naming it `overrides`/`subtests` instead of `configs`? Configs is very generic and it's not obvious we're talking about TestOptions/test.toml here.

Reply:
My point is that it's not obvious from the test reader's POV. If you look at any `test.toml`, it's not obvious what `[configs._]` is, it can be any config. But if it was named e.g. `subtests` or `tests.overrides`, it would be more clear that we're defining multiple tests.